data/paradise world jobs.txt

```diff
@@ -403,3 +403,92 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+
+# Brig bulk missions
```

Comment:
This comment suggests that the missions below should require a brig, but I don't see such requirement in the missions below. (I would be fine with not requiring a brig, but then change the comment here.)

Reply:
Is the comment even necessary? There are no such comments elsewhere in the file other than what is also added by this PR.